src/sass/modules/_m-alert.scss

```diff
@@ -17,7 +17,7 @@
 }
 
 .va-alert-body {
-  padding-left: 2.5rem;
+  padding-left: 3.5rem;
```

Comment:
It does solve the issue though. `padding-right` would push the icon away from the edge of the box.